src/vehicle_use.cpp

```diff
@@ -811,23 +813,29 @@ void vehicle::use_controls( const tripoint &pos )
         }
     }
 
-    if( has_part( "TURRET" ) ) {
-        options.emplace_back( _( "Set turret targeting modes" ), keybind( "TURRET_TARGET_MODE" ) );
+    //  TODO: Decide - Do I disable options or remove them entirely?
+    if( has_part( "TURRET" ) /*&& !player_character.has_trait(trait_BRAWLER) */ ) {
```

Comment:
Maybe add the reason to the disabled entry? E.g. `"Aim turrets manually (disabled because of trait)"`

An option that is suddenly missing for no apparent reason may look like a bug.

Reply:
I'm having a bit of a c++-brainfart moment. How do avoid writing that `"Aim turrets manually (disabled because of trait)"` + an ifelse check for brawler trait for each `options.emplace_back`? :D